os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -160,6 +160,11 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define MPU_REGION_ADDRESS_ALIGNMENT 32	/* MPU regions must be 32 byte address aligned */
+#define MPU_REGION_MINIMUM_SIZE 32		/* Min MPU region size is 32 bytes */
```

Comment:
If it is for stack guard, I think we can give this as config.

Reply:
yes added in config